eth_pydantic_types/string.py

```diff
@@ -0,0 +1,70 @@
+from typing import Any, ClassVar, Optional, Tuple, Type
+
+from pydantic_core.core_schema import (
+    CoreSchema,
+    ValidationInfo,
+    str_schema,
+    with_info_before_validator_function,
+)
+
+from eth_pydantic_types.hashing import get_hash_examples, get_hash_pattern
+from eth_pydantic_types.hex import BaseHexStr
+from eth_pydantic_types.validators import validate_str_size
+
+
+class String(BaseHexStr):
```

Comment:
it seems like this is a `HexStr` and not a `String` (sorry made this comment already but just assuring)